src/device/device_ledger.cpp

```diff
@@ -127,7 +127,7 @@ namespace hw {
     }
 
     bool operator==(const crypto::key_derivation &d0, const crypto::key_derivation &d1) {
-      return !memcmp(&d0, &d1, sizeof(d0));
+      return !crypto_verify_32(&d0, &d1, sizeof(d0));
```

Comment:
@moneromooo-monero I can't find an overload of `crypto_verify_32` that takes three parameters. Am I missing/misunderstanding something?

Reply:
Probably that this file isn't a dep of unit tests nor performance tests so didn't get rebuilt. Fixing.